### PR TITLE
My Code Review Suggestions

### PR DESCRIPTION
**Pull request by smith4ea**
Description of the project: 
This is a project to help you complete your tasks/goals

What I learned from this code review:
* I learned that the project did not have a proper ui implemented so I gave them a starter UI
* I learned about the xml design process
* I learned that to properly be able to do a code review you must provide ample code to the reviewer

Analysis of the program:
-the project does not have much relevant code in it to begin going towards the goal of the app
-the program was available in GitHub midday Sunday
-did not see any comments
-the program compiled but would not run on the emulator
-I suggested these changes because they give them a basis to progress towards the completion of their application

### CODE REVIEW

File: app/src/main/java/com/example/waifu/MainActivity.kt [hunk 1]
```diff
@@ -7,12 +7,20 @@ import android.view.View
...
 class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        var taskGoalName = editTextName.text
+
+        button_confirm.setOnClickListener {
+            editTextName.text.clear()
```
Comment (smith4ea) [inline app/src/main/java/com/example/waifu/MainActivity.kt:22]: Added storing the text from the editText box allowing to store the name of your goal/task as well as clearing out the box when the button is clicked

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -1,9 +1,51 @@
...
+        android:ems="10"
+        android:inputType="textPersonName"
+        android:text="Name"
+        tools:layout_editor_absoluteX="97dp"
+        tools:layout_editor_absoluteY="254dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/textView_Title"
+        app:layout_constraintVertical_bias="0.112" />
```
Comment (smith4ea) [inline app/src/main/res/layout/activity_main.xml:49]: added the proper UI elements that were missing

File: app/src/main/java/com/example/waifu/MainActivity.kt (hunk 1 above)
Comment (discospiff) [reply app/src/main/java/com/example/waifu/MainActivity.kt:22]: Is this used anywhere?  If not, it's an incomplete feature.

**Comment by discospiff** [comment]
It looks like this is worth merging, as the group already merged it.  :)

**Pull request merged**